Week3/homework/js-exercises/creditCardProject.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+function validateCreditNumber(num) {
+    const array = [];
+    for(let i = 0; i < num.length; i++){
+        array.push(num[i]);
```

Comment:
You don't need to convert a string to an array. A string is accessible with index similar to an array.

Reply:
well, maybe not a bad idea if you want to iterate over them using `filter`